src/shared/utils/isFirstChild.ts

```diff
@@ -0,0 +1,6 @@
+export const isFirstChild = (el: Element | null) => {
+  if (!el) return false;
+  return (
+    el.parentElement && Array.from(el.parentElement.children).indexOf(el) === 0
+  );
+};
```

Comment:
What's the point of this?

Reply:
Same as this: https://github.com/Light-it-labs/fe-boilerplate/pull/42#discussion_r1869849228

But in this case choosing the first date segment from the field to focus when it's invalid.